CMakeLists.txt

```diff
@@ -222,4 +249,16 @@ endif()
 
 if(CUKE_ENABLE_EXAMPLES)
     add_subdirectory(examples)
+
+    #check if c++11 is nedded
+    if(${Qt5Core_FOUND})
+        find_program(QMAKE_EXECUTABLE NAMES qmake HINTS ${QTDIR} ENV QTDIR PATH_SUFFIXES bin)
+        execute_process(COMMAND ${QMAKE_EXECUTABLE} -query QT_VERSION OUTPUT_VARIABLE QT_VERSION)
+        if(QT_VERSION GREATER 5.6.99)
```

Comment:
Please use this instead:
```cmake
if(NOT QT_VERSION VERSION_LESS 5.7)
```

That does a proper lexicographic comparison of the version number.